src/audio/src/src.c

```diff
@@ -456,15 +470,27 @@ static struct comp_dev *src_new(const struct comp_driver *drv,
 {
 	struct comp_dev *dev;
 	struct comp_data *cd;
-	struct ipc_config_src *ipc_src = spec;
+#if CONFIG_IPC_MAJOR_4
+	struct ipc4_config_src *ipc_src = spec;
 
+	dcache_invalidate_region(spec, sizeof(*ipc_src));
```

Comment:
Data from IPC should already be invalidated by this point in the IPC logic, this means we should not have to do it for all modules. Please check though if you added this due to a bug.

Reply:
At the moment looks like it is not invalidated
added PR #5215 about that